ci-operator/step-registry/openshift/e2e/cert-rotation/shutdown/openshift-e2e-cert-rotation-shutdown-commands.sh

```diff
@@ -88,6 +91,9 @@ set -x
 # Check that time on nodes has been updated
 until run-on-all-nodes "timedatectl status"; do sleep 30; done
 
+# Restart kubelet
+run-on-all-nodes "systemctl restart kubelet"
```

Comment:
Is this restart for time skew?

Reply:
Yes, I found it sometimes helps (although it might need to be reverted eventually)